lib/GLPI/Agent/Task/Inventory/Generic/Remote_Mgmt/RMS.pm

```diff
@@ -0,0 +1,74 @@
+package GLPI::Agent::Task::Inventory::Generic::Remote_Mgmt::RMS;
+
+use strict;
+use warnings;
+
+use parent 'GLPI::Agent::Task::Inventory::Module';
+
+use English qw(-no_match_vars);
+use UNIVERSAL::require;
+
+use GLPI::Agent::Tools;
+
+sub isEnabled {
+    my (%params) = @_;
+
+    if (OSNAME eq 'MSWin32') {
+
+        GLPI::Agent::Tools::Win32->use();
+
+        my $key = getRegistryKey(
+            path => 'HKEY_LOCAL_MACHINE/SOFTWARE/Usoris/Remote Utilities Host/Host/Parameters',
+            # Important for remote inventory optimization
+            required    => [ qw/InternetId/ ],
+            maxdepth    => 1,
+            logger => $params{logger}
+        );
+
+        return 1;
+    }
+
+    return 0;
```

Comment:
We missed `$key` is not used here. I would like also simplify the code until you provide a process to check this on other platform:
```suggestion
    return 0 unless OSNAME eq 'MSWin32';

    GLPI::Agent::Tools::Win32->use();

    my $key = getRegistryKey(
        path => 'HKEY_LOCAL_MACHINE/SOFTWARE/Usoris/Remote Utilities Host/Host/Parameters',
        # Important for remote inventory optimization
        required    => [ qw/InternetId/ ],
        maxdepth    => 1,
        logger => $params{logger}
    );

    return defined($key) ? 1 : 0;
```